.gitignore

```diff
@@ -2,3 +2,4 @@
 npm-debug.log
 node_modules
 package-lock.json
+.idea/
```

Comment:
If not used please remove `.idea`

Reply:
`.idea` contains the project specific settings for project for Intellij IDE & I don't what to add it in the project.